stock_available_portal/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module adds the functionality to display **Products** in the portal.
+Access rights are defined in the **Settings**.
```

Comment:
This module adds the functionality to display **Products** in  portal.
Configurator domains are used define which products are visible and which users have access to products